src/extension/monitor/hibernate/src/main/resources/org/geoserver/monitor/hib/db.properties

```diff
@@ -20,3 +20,5 @@ url=jdbc:h2:file:${GEOSERVER_DATA_DIR}/monitoring/monitoring
 #url=jdbc:oracle:thin:@[host]:[port]\:[instance]
 #username=[username]
 #password=[password]
+
+gwc.context.suffix=gwc
```

Comment:
For this one I'd rather fix the source issue (more on this later, let me have a look)

Reply:
Can we remove this property? Should still work